app/services/event_log/move_runner.rb

```diff
@@ -16,7 +16,6 @@ def call
         when Event::APPROVE
           move.status = Move::MOVE_STATUS_REQUESTED
           move.date = event.date
-          move.determine_supplier
```

Comment:
now that we are live, we determine the supplier on move-creation and then it never changes on move-update